manifests/puppet.pp

```diff
@@ -35,10 +35,9 @@
 
   if $deploy {
     file { "${certs::pki_dir}/puppet":
-      ensure  => directory,
-      owner   => 'puppet',
-      mode    => '0700',
-      require => Class['puppet::server::install'],
```

Comment:
Unfortunately, this exists because Puppet 4 packages do not create a `puppet` user (unless it's been fixed) in the RPM so we aren't guaranteed this will work at the time of calling without requiring this class here.